example/src/utils/index.tsx

```diff
@@ -71,6 +71,9 @@ const buildDefaultApiClientConfiguration = (
         exponentialBackoffBase: 2,
         exponentialBackoffScale: 0.5,
     };
+    const requestAdapterConfiguration = {
```

Comment:
@migbot This is already declared as parameter in line 55. This was moved to parameter so that other clients can pass a different value.